test/mobile/op_deps/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(op_deps_project)
+
+# Find torch library
+add_library(torch UNKNOWN IMPORTED)
+find_package(Torch REQUIRED)
+include_directories(${TORCH_INCLUDE_DIRS})
+link_directories(${TORCH_LIBRARIES})
```

Comment:
If this ever turns into a real tutorial example that people will copy paste off of, it will be better to use `target_include_directories` to encourage people to write it in the modern cmake style